[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:190]Enter CLIENT_SetAutoReconnect:[cbAutoConnect=0x7f4473768ac8, dwUser=(nil)].
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:192]Leave CLIENT_SetAutoReconnect.
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:210]Enter CLIENT_SetConnectTime:[nWaitTime=5000, nTryTimes=1].
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:223]Leave CLIENT_SetConnectTime.
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:234]Enter CLIENT_SetNetworkParam:[nWaittime=0, nConnectTime=10000, nConnectTryNum=0, nSubConnectSpaceTime=0, nGetDevInfoTime=0, nConnectBufSize=0, nGetConnInfoTime=3000, nSearchRecordTime=0, nsubDisconnetTime=0, byNetType=0, byPlaybackBufSize=0, bDetectDisconnTime=0, bKeepLifeInterval=0, nPicBufSize=0].
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:249]Leave CLIENT_SetNetworkParam.
[2020-09-27T08:55:15.328 info Netsdk:41296 8611:8633 dhnetsdk.cpp:586]Enter CLIENT_LoginEx2. IP:192.168.25.109, port:37777, SpeCap:0, lpDeviceInfo:0x7f43e48a3cd0.
[2020-09-27T08:55:15.328 error Netsdk:41296 8611:8633 TPTCPClient.cpp:423]connect() failed: Operation now in progress
[2020-09-27T08:55:15.678 info Netsdk:41296 8611:8633 dhnetsdk.cpp:622]Leave CLIENT_LoginEx2. error:0, login ID:139929565136912.
[2020-09-27T08:55:15.679 info Netsdk:41296 8611:8633 dhnetsdk.cpp:7400]Enter CLIENT_SetSnapRevCallBack. [OnSnapRevMessage=0x7f4473768b00, dwUser=(nil).]
[2020-09-27T08:55:15.679 info Netsdk:41296 8611:8633 dhnetsdk.cpp:7403]Leave CLIENT_SetSnapRevCallBack.
[2020-09-27T08:55:15.679 info Netsdk:41296 8611:8633 dhnetsdk.cpp:7440]Enter CLIENT_SnapPictureEx. [lLoginID=139929565136912, par=0x7f43e40655d0, reserved=0x7f43e4063fa0.]
[2020-09-27T08:55:15.679 info Netsdk:41296 8611:8633 dhnetsdk.cpp:7460]Leave CLIENT_SnapPictureEx. ret:1.
